### PR TITLE
Setup publishing via GitHub Actions.

### LINKED ISSUE #13: Publish latest to @Netflix/eslint-config-netflix
Title says it all — #11 describes install steps that currently do not work:

```
npm install --save-dev @Netflix/eslint-config-netflix
```

Older versions can still be installed via:

```
npm install --save-dev eslint-config-netflix
```

### PR DESCRIPTION
**Pull request by theengineear**
See “PUBLISHING.md” for more details.

Closes #13.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "eslint-config-netflix",
+  "name": "@netflix/eslint-config-netflix",
```
Comment (theengineear) [inline package-lock.json:2]: Looks like the scope prefix wasn’t previously added. Just committing the result of running `npm install` here.

File: bump.sh
Comment (theengineear) [inline bump.sh:1]: This is the “assisted” publish method noted in `PUBLISHING.md`. You can 100% do these steps manually as documented, but I find it easier to do a one-step publish that tees to both NPM and JSR.

File: deno.json
Comment (theengineear) [inline deno.json:1]: This is the thing JSR wants.

File: package-lock.json
Comment (theengineear) [inline package-lock.json:1]: Looks like the scope prefix wasn’t previously added. Just committing the result of running `npm install` here.

File: bump.sh
Comment (theengineear) [inline bump.sh:1]: This is the “assisted” publish method noted in `PUBLISHING.md`. You can 100% do these steps manually as documented, but I find it easier to do a one-step publish that tees to both NPM and JSR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 #### Features
 
+* Support for publishing to NPM / JSR via GitHub Actions.
```
Comment (klebba) [inline CHANGELOG.md:6]: nit: move below the more important item

**Review by klebba — APPROVED** [review]
LG2M

**Pull request merged**